docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -1210,6 +1210,27 @@ SELECT
 FROM table
 ```
 
+#### Vararg and kwarg arguments
+Varargs (variable arguments) and kwargs (keyword arguments) are possible to use in Python macros definitions. Kwargs can be used in SQL with the property assignment operator `:=`. 
```

Comment:
Maybe we should expand a little on what "varargs" and "kwargs" mean and perhaps show a small python function as an example to establish the connection between the two.